src/service/hedera-validator.ts

```diff
@@ -0,0 +1,13 @@
+import { externalRequest } from "./request";
+
+export const metadataEndpoint =
+    "http://localhost:6200/api/v1/metadata?gasPriceGwei=";
```

Comment:
It would be better to extract the whole metadata endpoint in `vue.config`, or at least the url.

Reply:
Done!